src/fut/mod.rs

```diff
@@ -44,7 +44,38 @@ use actor::Actor;
 /// Trait for types which are a placeholder of a value that may become
 /// available at some later point in time.
 ///
-/// This is similar to `futures::Future` trait, except it works with `Actor`
+/// `ActorFuture` is very similar to a regular `Future`, only with subsequent combinator closures accepting the relevant actor and its context, in addition to the result.
+///
+/// `ActorFuture` allows for use cases where future processing requires access to the actor ot its context.
+///
+/// Here is an example of a handler on a single actor, deferring work to another actor, and
+/// then updating the initiating actor's state:
+///
+/// ```rust,ignore
+/// impl Message for SomeMessage {
+///     type Result = ();
+/// }
+///
+/// impl Handler<DeferredWork> for OriginalActor {
+///     type Result = ResponseActFuture<Self, (), Error>;
+///
+///     fn handle(&mut self, _msg: Refresh, ctx: &mut Context<Self>) -> Self::Result {
+///         let send_to_other = self.other_actor_addr
+///             .send(OtherMessage::new())
+///             .map_err(Error::from);
+///
+///         let update_self =
+///             wrap_future::<_, Self>(send_to_other).map(|result, actor, _ctx| {
+///                 // Actor's state updated here
+///                 actor.inner_state.update_from(result);
+///             });
+///         // return the wrapping future
+///         Box::new(returned_future);
+///     }
+/// }
+/// ```
+///
+/// See also [into_actor](trait.WrapFuture.html#tymethod.into_actor), which is similar in functionality
```

Comment:
`which provides future conversion using trait` sounds a bit more correct, as functionality is identical :)